olm-catalog/serverless-operator/project.yaml

```diff
@@ -44,7 +44,7 @@ dependencies:
   operator: 1.2.2
   # Previous versions required for downgrade testing
   previous:
-    serving: 1.2.0
-    eventing: 1.2.1
+    serving: 1.3.0
```

Comment:
This is temporary as we are going to bump the versions anyway. Otherwise downgrade tests will fail.

Reply:
Sounds good, this middle step is a new step to every bump moving forward, right? Maybe we can add a comment like (line 4)

```
# For minor and major version bumps, bump all `dependenceis.previous` to whatever `dependencies` has set.
version: 1.25.0

Reply:
Added